### PR TITLE
[test_badmicoctrl_1]:pr_test_with_helloworld

### PR DESCRIPTION
**Pull request by badmicroctrl**
hello world

Author: badmicroctrl@163.com

### What?
for paticipating the programme , submit a pr

### What changes are included in this PR?
include a helloworld code

### Are these changes tested?
tested,result as follow:

### CODE REVIEW

**Comment by Light-City** [comment]
Are these changes tested?
tested

this answer is so bad.

**Comment by Light-City** [comment]
1.The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

2.your pr title style is wrong.

**Comment by badmicroctrl** [comment]
> 1.The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md
> I corrected the directory structure.
> 2.your pr title style is wrong.

modified

**Comment by Light-City** [comment]
reject.

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by badmicroctrl** [comment]
> The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

I modify the directory structure again, please check whether this time is ok , if ok I can enter the group...

**Comment by martinx** [comment]
The structure is still incorrect, you can refer to the following :

```
-- 团队名称/   (目录，小写字母和下划线)
    README.md    (文件，团队介绍）
    --- 个人名称   (目录，小写字母和下划线)
       ---- src        （源文件目录）
            main.cpp   (Hello World 的演示代码)
       RADME.md    (文件，个人文档，记录等）

```

**Comment by badmicroctrl** [comment]
> The structure is still incorrect, you can refer to the following :
> 
> ```
> -- 团队名称/   (目录，小写字母和下划线)
>     README.md    (文件，团队介绍）
>     --- 个人名称   (目录，小写字母和下划线)
>        ---- src        （源文件目录）
>             main.cpp   (Hello World 的演示代码)
>        RADME.md    (文件，个人文档，记录等）
> ```

I rectify the files according to your guidance, please check ,thank you

File: badmicroctrl_team/badmicroctrl/src/helloworld.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
+// Created by badmicroctrl on 2023/6/17.
+//
+
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
```
Comment (martinx) [inline badmicroctrl_team/badmicroctrl/src/helloworld.cpp:8]: Indent 2 spaces

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**